mantidproject/mantid
# Separate q keys for ISIS SANS user and batch file

**Description of work.**
This PR provides a separate q key to the ISIS SANS batch file so that searching for a new batch file will automatically open up the directory of the last used batch file, rather than the last used user file

**To test:**
1. Save the user and batch files in the attached data in different directories
2. Interfaces -> SANS -> ISIS SANS
3. Click load user file and select the attached user file
4. Click load batch file and select the attached batch file
5. Reselect the user file. Confirm that clicking `Load User File` opens a dialog box in the directory *of the user file*
6. Reselect the batch file. Confirm that clicking `Load Batch File` opens a dialog box in the directory *of the batch file*

Fixes #25536 

**Data:**
[loqPRData.zip](https://github.com/mantidproject/mantid/files/3111393/loqPRData.zip)


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
